gcp_variant_transforms/libs/bigquery_vcf_schema_converter.py

```diff
@@ -203,19 +206,20 @@ def generate_header_fields_from_schema(schema):
     if field.name in _NON_INFO_OR_FORMAT_CONSTANT_FIELDS:
       continue
     elif field.name == bigquery_util.ColumnKeyConstants.CALLS:
-      _add_format_fields(field, formats)
+      _add_format_fields(field, allow_incompatible_schema, formats)
     else:
-      _add_info_fields(field, infos)
+      _add_info_fields(field, allow_incompatible_schema, infos)
 
   return vcf_header_io.VcfHeader(infos=infos, formats=formats)
 
 
-def _add_format_fields(schema, formats):
-  # type: (bigquery.TableFieldSchema, Dict[str, _Format]) -> None
+def _add_format_fields(schema, allow_incompatible_schema, formats):
```

Comment:
Yeah, the `input` and `output` are the main concerns that stopping me from setting `allow_incompatible_schema=False`. The `input_configuration` clarifies my concern. Thank you so much for explaining this!

Reply:
Thanks for making the change! I should have clarified that the "inputs" before "outputs" convention is not really a "rule" :) I just borrowed the concept from my C++ days and it made the code much easier to read as we had a consistent ordering of arguments across the codebase. I just thought that it would be helpful to use this ordering wherever makes sense in our code too.